tests/interpolation/test_custom_resolvers.py

```diff
@@ -345,44 +346,28 @@ def test_clear_cache(restore_resolvers: Any) -> None:
     assert old != c.k
 
 
-@mark.parametrize(
-    ("cfg", "expected"),
-    [
-        ({"a": 0, "b": 1}, {"a": 0, "b": 1}),
-        ({"a": "${y}"}, {"a": -1}),
-        ({"a": 0, "b": "${x.a}"}, {"a": 0, "b": 0}),
-        ({"a": 0, "b": "${.a}"}, {"a": 0, "b": 0}),
-        ({"a": "${..y}"}, {"a": -1}),
-    ],
-)
-def test_resolver_output_dict_to_dictconfig(
-    restore_resolvers: Any, cfg: Dict[str, Any], expected: Dict[str, Any]
-) -> None:
-    OmegaConf.register_new_resolver("dict", lambda: cfg)
+@mark.parametrize("readonly", [True, False, None])
```

Comment:
I think we can go with True and False only. I know that in other places we are also testing None but it's probably redundant in most of them.

Reply:
Done in 55f8041